docs/user-guide.md

```diff
@@ -24,26 +24,26 @@ that might be null then you write `@Nullable String x;`.
 
 ## Types and nullness
 
-In the JSpecify world, most references have one of three possible properties
+In the JSpecify world, references have one of three possible properties
 regarding nullness:
 1. They can be null.
 2. They can't be null.
 3. We don't know if they can be null or not.
 
-If `x` is a reference of a certain type, then if `x` can be null, `x.toString()` is
+If `x` is a reference of a certain type, then if `x` can be null, the dereference `x.toString()` is
 unsafe because it could produce a NullPointerException. If `x` can't be null,
 `x.toString()` can never produce a NullPointerException. And if we don't know
 whether `x` can be null or not, `x.toString()` might or might not be unsafe.
 
 There are two JSpecify annotations that indicate these properties:
 * `@Nullable` applied to a type denotes a reference of that type that can be null.
-* `@NullMarked` applied to a module, class, or package means that a reference in
+* `@NullMarked` applied to a module, package, or class means that a reference in
```

Comment:
Follow-up to https://github.com/jspecify/jspecify/pull/201#discussion_r663064798

I switched the order in this document. Should we do the same throughout the javadoc, e.g. here: https://github.com/jspecify/jspecify/blob/8797899cff85804cd3cf049854e498805b122104/src/java9/java/org/jspecify/nullness/NullMarked.java#L28

Reply:
I think that'll get fixed naturally when we come by and add "method" (which I seem to recall looking like what we want to do, when last I saw the issue).